sdk/mediaservices/Azure.Media.Analytics.Edge/README.md

```diff
@@ -0,0 +1,207 @@
+- Programatically construct media graph topologies and instances
+
+[Product documentation][doc_product] | [Direct methods][doc_direct_methods] | [Media graphs][doc_media_graph] | [Source code][source] | [Samples][samples]
+
+## Getting started
+
+This is a models only sdk. All client operations are done using the [Microsoft Azure IoT SDKs](https://github.com/azure/azure-iot-sdks). This sdk provides models you can use to interact with the Azure Iot SDKs.
+
+### Authenticate the client
+
+As mentioned above the client is coming from Azure IoT SDK. You will need to obtain an [IoT device connection string][iot_device_connection_string] in order to authenticate the Azure IoT SDK. For more information please visit: https://github.com/Azure/azure-iot-sdk-csharp. 
+```C# Snippet:Azure_Search_Samples_ConnectionString
+var connectionString = "connection-string";
+this._serviceClient = ServiceClient.CreateFromConnectionString(connectionString);
+```
+
+### Install the package
+
+Install the Live Video Analytics client library for .NET with NuGet:
+
+`dotnet add package Azure.Media.Analytics.Edge --version 1.0.0-beta.1`
+
+### Prerequisites
+
+* C# is required to use this package.
+* You need an active [Azure subscription][azure_sub], and an [IoT device connection string][iot_device_connection_string] to use this package.
+* You will need to use the version of the SDK that corresponds to the version of the LVA Edge module you are using.
+
+    | SDK  | LVA Edge Module  |
+    |---|---|
+    | 1.0.0b1  | 2.0  |
+
+### Creating a graph topology and making requests
+Please visit the [Examples](#examples) for starter code
+## Key concepts
+
+### MediaGraph Topology vs MediaGraph Instance
+A _graph topology_ is a blueprint or template of a graph. It defines the parameters of the graph using placeholders as values for them. A _graph instance_ references a graph topology and specifies the parameters. This way you are able to have multiple graph instances referencing the same topology but with different values for parameters. For more information please visit [Media graph topologies and instances][doc_media_graph] 
+
+### CloudToDeviceMethod
+
+The `CloudToDeviceMethod` is part of the [azure-iot-hub SDk][iot-hub-sdk]. This method allows you to communicate one way notifications to a device in your IoT hub. In our case, we want to communicate various graph methods such as `MediaGraphTopologySetRequest` and `MediaGraphTopologyGetRequest`. To use `CloudToDeviceMethod` you need to pass in two parameters: `method_name` and `payload`. 
+
+The first parameter, `method_name`, is the name of the media graph request you are sending. Make sure to use each method's predefined `method_name` property. For example, `MediaGraphTopologySetRequest.method_name`. 
+
+The second parameter, `payload`, sends the entire serialization of the media graph request. For example, `MediaGraphTopologySetRequest.serialize()`
```

Comment:
I don't think we are passing the payload as a parameter in the example below. We are using the SetPayloadJson instead.